README.md

```diff
@@ -324,3 +324,47 @@ Please refer to mocha documentation to know how to use them.
 
 If you want to use that with storybook, you'll need to add them to your mocha config and storybook config files.
 You can find the complete configuration by looking at the [samples directory](https://github.com/mthuret/storybook-addon-specifications/tree/master/.storybook)
+
+#### Loading External Test Files
+
+It is also possible to load your test files externally from their respective test files.
+
+```
+|- example.stories.js
+|- example.test.js
+|- example.js
+```
+
+This allows us to run both our test runner, and view the test results inside of React Storybook.
+
+```js
+import React from 'react'
+import { storiesOf } from '@kadira/storybook'
+import { specs } from 'storybook-addon-specifications'
+
+import { tests } from './Example.test'
+import Example from './example'
+
+storiesOf('Example', module)
+  .add('Default', () => {
```

Comment:
What I did was add a Decorator, like this:
```js
  stories.addDecorator((story) => {
    specs(() => require('./Component.test'));
    return story();
  })
```

That way, when adding a lot of stories, all tests appear with all stories. I assume this is what you want when extracting tests into a different file.